db/migrate/20160617194853_add_details_to_recipients.rb

```diff
@@ -0,0 +1,9 @@
+class AddDetailsToRecipients < ActiveRecord::Migration
+  def change
+    add_column :recipients, :age, :integer
```

Comment:
💭 I've usually used `date_of_birth` as the field in the database and then calculated the age. That way the value for age doesn't have to be updated manually. I can see why we could keep this as age, which would represent "Age of recipient on (campaign name)". This would also mean we are asking for less personal information, which would be a plus. I'm cool with keeping this `age` but thought I'd provide an alternative